ansible/inventory/production/group_vars/all/users.yml

```diff
@@ -0,0 +1,21 @@
+# Login, SSH public key, comment and state of users with admins rights
+# State absent removes the user account
+adm_acct:
+  andz:
```

Comment:
Now in order for me to add mine this needs to be a list instead, right? Could you add the barebones for my account, please?
The user can be `core`.

Reply:
I have added documentation and examples on how to add a user and grant admin privileges in:
https://github.com/PrivacyLx/devops/pull/21/files#diff-04c6e90faac2675aa89e2176d2eec7d8R55

Let me know if this doesn't work for you.